e2e/README.md

```diff
@@ -0,0 +1,46 @@
+# Rook Test Framework
+ 
+The Rook Test Framework is a Go project that that uses a Docker container with Kubernetes
+tools and Docker pre-installed to quickly create a 3 node Kubernetes cluster using Docker in 
+Docker (DinD) techniques, installs Rook and then runs automation tests to verify its functionality.
+ 
+##Requirements
+1. Docker version => 1.2 && < 17.0
+2. Ubuntu Host 16 (currently the framework has only been tested on this version)
+ 
+##Instructions 
+On your Ubuntu box do the following;
+1.   Be sure to allocate atleast 4 processors to your vm
+2.   Navigate to the rook directory of the clone sources and enter the following;
```

Comment:
what about this wording? 
"Navigate to the root directory of the repo and run the following:"